src/modelio.cs

```diff
@@ -227,12 +227,20 @@ interface MDLAsset : NSCopying
 		[Export ("objectAtIndex:")]
 		MDLObject GetObject (nuint index);
 
+		[Deprecated (PlatformName.iOS, 15, 0, message: "Use the 'Originals' instead.")]
+		[Deprecated (PlatformName.TvOS, 15, 0, message: "Use the 'Originals' instead.")]
+		[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use the 'Originals' instead.")]
+		[Deprecated (PlatformName.MacCatalyst, 15, 0, message: "Use the 'Originals' instead.")]
```

Comment:
sounds weird - `"Use 'Originals' instead."` or `"Use the 'Originals' property instead."`

Reply:
I can change it to the 'property' one